Rewrite Content-Digest and Repr-Digest examples with real, verifiable… (#45076)
mdn/content: files/en-us/web/http/reference/headers/content-digest/index.md

## Examples

### A SHA-256 Content-Digest in a response

A user-agent requests a resource:

```http
GET /items/123 HTTP/1.1
Host: example.com
```

The server responds with a `Content-Digest` of the message content using the SHA-256 algorithm.

> The digest is calculated over the exact bytes of the message body, `{"hello": "mdn"}` (16 bytes, with no trailing line break):

Reviewer: is this more clear?
```suggestion
The digest is calculated over the exact bytes of the message body, `{"hello": "mdn"}` (16 bytes, explicitly not including any trailing line break):
```
